sdl_android_lib/src/com/smartdevicelink/abstraction/constants/SDLVersion.java

```diff
@@ -0,0 +1,7 @@
+package com.smartdevicelink.abstraction.constants;
+
+public enum SDLVersion {

```

Comment:
We're not accepting anything that differentiates between Ford specific head units